src/dbus2mqtt/dbus/dbus_client.py

```diff
@@ -809,12 +827,77 @@ async def _on_mqtt_msg(self, msg: MqttMessage):
 
                                         try:
                                             logger.info(f"on_mqtt_msg: property={property.property}, value={payload_value}, bus_name={bus_name}, path={path}, interface={interface_config.interface}")
+                                            # converted_property_value = convert_connman_property_args(payload_value)
                                             await self.set_dbus_interface_property(interface, property.property, payload_value)
+                                            
+                                            # Send property set response if configured
+                                            await self._send_mqtt_response(
+                                                interface_config, f"set_{property.property}", [payload_value],
+                                                payload_value, None, bus_name, path
+                                            )
+                                            
                                         except Exception as e:
                                             logger.warning(f"on_mqtt_msg: property={property.property}, value={payload_value}, bus_name={bus_name} failed, exception={e}")
+                                            
+                                            # Send property set error response if configured
+                                            await self._send_mqtt_response(
+                                                interface_config, f"set_{property.property}", [payload_value],
+                                                None, e, bus_name, path
+                                            )
 
         if not matched_method and not matched_property:
             if payload_method:
                 logger.info(f"No configured or active dbus subscriptions for topic={msg.topic}, method={payload_method}, bus_name={payload_bus_name}, path={payload_path or '*'}, active bus_names={list(self.subscriptions.keys())}")
             if payload_property:
                 logger.info(f"No configured or active dbus subscriptions for topic={msg.topic}, property={payload_property}, bus_name={payload_bus_name}, path={payload_path or '*'}, active bus_names={list(self.subscriptions.keys())}")
+
+    async def _send_mqtt_response(self, interface_config, method_name: str, method_args: list[Any], 
+                             result: Any, error: Exception | None, bus_name: str, path: str):
+        """Send MQTT response for a method call if response topic is configured"""
+    
+        if not interface_config.mqtt_response_topic:
+            return
+        
+        try:
+            # Build response context
+            response_context = {
+                "bus_name": bus_name,
+                "path": path,
+                "interface": interface_config.interface,
+                "method": method_name,
```

Comment:
See previous comment about distinction between methods and properties. If its a response following a method call this is ok, if it is a response from a property call I would expect that `method` and `method_args` are not part of the response and instead a `property` key is used.

This also aligns with the request parameters where it's either

* method
* args

or 

* property
* value